data/remnant/remnant ships.txt

```diff
@@ -265,6 +265,206 @@ ship "Albatross" "Albatross (Sniper II)"
 	turret
 	turret "Point Defense Turret"
 
+
+ship "Modified Dromedary"
+	plural "Modified Dromedaries"
+	sprite "ship/modified dromedary"
+	thumbnail "thumbnail/modified dromedary"
+	attributes
+		licenses
+			"Remnant Capital"
+		category "Utility"
+		"cost" 57600000
+		"shields" 15100
+		"hull" 9600
+		"required crew" 95
+		"bunks" 300
+		"mass" 2140
+		"drag" 15.1
+		"heat dissipation" .4
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"weapon capacity" 270
+		"engine capacity" 195
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		"asteroid scan power" 42
+		"atmosphere scan" 100
+		weapon
+			"blast radius" 260
+			"shield damage" 2600
+			"hull damage" 1300
+			"hit force" 3900
+	outfits
+		"Thrasher Turret" 2
+		"Point Defense Turret" 2
+
+		"Aeon Cell"
+		"Crystal Capacitor" 2
+		"Thermoelectric Cooler" 3
+		"Quantum Key Stone"
+		"Salvage Scanner"
+		"Tuning Rifle" 95
+
+		"Smelter-Class Thruster"
+		"Smelter-Class Steering"
+		"Hyperdrive"
+		"Scram Drive"
+
+	engine 32.5 207
+	engine 3 207
+	engine -27.5 207
+	gun 65 -111
+		angle 40
+	gun -72 -97.5
+		angle 320
+	gun -66.5 61
+		angle 270
+	gun 78 64
+		angle 90
+	turret 10 -136.5 "Thrasher Turret"
+	turret -21.5 -57.5 "Point Defense Turret"
+	turret 51 101
+	turret -63.5 120 "Thrasher Turret"
+	turret 52.5 143 "Point Defense Turret"
+	bay "Fighter" -51 -54 under
+	bay "Fighter" 47 -40 under
+	leak "leak" 30 50
+	leak "flame" 30 80
+	leak "big leak" 50 30
+	explode "small explosion" 25
+	explode "medium explosion" 35
+	explode "large explosion" 45
+	explode "huge explosion" 30
+	"final explode" "final explosion large"
+	description `The Tycho Crater Cooperative's Dromedary was one of the foremost colonization ships during the early era before piracy became rampant and triggered the necessity for city-ships to be more combat-oriented. These ships incorporate extensive hydroponics, large construction equipment installations, and hundreds of densely packed "dream cells" built to transport people in bulk.`
+	description "	Since the Remnant do not have schematics for the Dromedary, just the highly-modified example of the Shattered Light, this ship is actually a commemorative replica of that specific ship. As such, it combines the original's odd mix of eclectic modifications with new Remnant systems."
+	description "	Unfortunately, since they lack the manufacturing facilities that would normally support a large construction project, it will cost massively more than it would have originally back on Luna."
```

Comment:
Why couldn't First Last go ask the Luna shipyard for their old schematics? For that matter, why didn't the first Remnant colonizers have schematics for one of their city ships they presumably intended on keeping working for quite a while without visiting an established shipyard. The latter could be IP law/industrial espionage prevention, but I doubt the modern day Tycho shipyard would object to giving First Last some centuries old ship designs, especially if they've done the Emerald Sword missions and are well known as being interested in old spaceships.

Of course, the Remnant still don't have the systems to mass-produce a Dromedary, even if they did have the schematics. I don't think the price should be changed, but maybe fetching schematics would lower the time-to-availability by a few days.

Reply:
Well, a couple things here:

For one, I wanted to leave open the possibility that the *original* dromedary might be out there. I don't see a real need for another large ship in human space right now, but it is theoretically a possibility that. 

For a second, the Remnant aren't really that interested in producing the original Dromedary. This modified version is significantly upgraded over the original, and the Remnant prefer this to the original anyway.

Thirdly, sounds like a good follow-up mission. Perhaps there are some aspects of its design that don't quite make sense to them; Maybe stuff has been twisted or distorted by its time adrift. Having schematics of its original construction could help them work out what has changed, too. Thanks for the idea!